CHANGELOG.md

```diff
@@ -2,8 +2,8 @@
 
 ## (4.0.0 IN PROGRESS)
 * Bump stripes to `v8`. Upgrade `react-redux` to `v8`. Refs UIPFPOL-54.
-
 * Support `inventory` interface version `13.0`. Refs UIPFPOL-52.
+* *BREAKING*: Update `@folio/stripes` to `8.0.0`. Refs UIPFPOL-53.
```

Comment:
It looks like the v8 update happened in #133. You don't really need two changelog entries for this work.